src/tee/components/MarkdownEditor.tsx

```diff
@@ -69,13 +72,37 @@ export function MarkdownEditor({
 
   const handleReady = handle.isReady();
 
+  const account = useCurrentAccount();
```

Comment:
PS: this hook is driving me crazy

Reply:
specifically, it's annoying that i have to check several levels of hierarchy (account, currentaccount, doc) and then i still have the type / anonymous stuff where if the type is anonymous i have to reinvent the name & a color every time...

my suggestion is that we set the name to anonymous (at least in the hook but i think in the doc too) and pick a random presence color and leave it on the contact doc.